README.md

```diff
@@ -196,76 +196,33 @@ import com.rudderstack.generated.*
 
 ### RudderStack JavaScript SDK
 
-- Import the RudderTyper-generated client using `require()` and make the calls if your framework supports them. Otherwise, you can use [**Browserify**](https://browserify.org/) to generate a bundle that supports your implementation. The implementation for each of the alternatives mentioned above will be as shown:
+There are two ways to get started with Rudder-Typer in your browser:
 
-#### Using the `require()`method
+#### 1. Using CDN (Snippet)
```

Comment:
Can we also point in the instructions at the end to the appropriate sample application for more details?

Reply:
I felt I didn't have to point this out. Let's do the same for NPM.

Reply:
Done